mapic/misttriggers/stream_buffer.go

```diff
@@ -82,12 +84,16 @@ func TriggerStreamBuffer(cli *config.Cli, req *http.Request, lines []string) err
 }
 
 type StreamHealthPayload struct {
-	StreamName string                  `json:"stream_name"`
-	SessionID  string                  `json:"session_id"`
-	IsActive   bool                    `json:"is_active"`
-	IsHealthy  bool                    `json:"is_healthy"`
-	Tracks     map[string]TrackDetails `json:"tracks,omitempty"`
-	Issues     string                  `json:"issues,omitempty"`
+	StreamName string `json:"stream_name"`
+	SessionID  string `json:"session_id"`
+	IsActive   bool   `json:"is_active"`
+
+	IsHealthy   bool   `json:"is_healthy"`
+	Issues      string `json:"issues,omitempty"`
+	HumanIssues string `json:"human_issues,omitempty"`
+
+	Tracks map[string]TrackDetails `json:"tracks,omitempty"`
+	Extra  map[string]interface{}  `json:"extra,omitempty"`
```

Comment:
I enjoy spelling this like
```suggestion
	Extra  map[string]any  `json:"extra,omitempty"`
```

Reply:
Yeah it was introduced with generics, `any` is just an alias for `interface{}`